libraries/chain/protocol_feature_manager.cpp

```diff
@@ -371,6 +371,20 @@ host function call will trigger a transition to the Savanna consensus algorithm.
 Builtin protocol feature: SYNC_CALL
 
 Enables synchronous calls to functions in other and own contracts.
+*/
+            {builtin_protocol_feature_t::savanna}
+         } )
+         (  builtin_protocol_feature_t::allow_non_canonical_signatures, builtin_protocol_feature_spec{
+            "ALLOW_NON_CANONICAL_SIGNATURES",
```

Comment:
I feel the `non` in the name causing some confusions when reading code together with `require_canonical_t::{yes, no}`. Like `if (!allow_non_canonical_signatures) use require_canonical_t::yes; ... `

Reply:
This is the clearest name I can think of for the protocol feature. Do you have a suggestion?

Reply:
fwiw I usually dislike "logic puzzle" names (which this one certainly classifies as :laughing: ) but my vote would be to keep it. Only other thing I could immediately think of is `NO_CANONICAL_SIGNATURE_CHECK` or `REMOVE_CANONICAL_SIGNATURE_CHECK` (at least these have only 1 positive/negative).

Reply:
`REMOVE_CANONICAL_SIGNATURE_CHECK` is better. But no need to change it now.